### ISSUE TITLE
location of the docs?

### ISSUE THREAD

**Issue by arsenovic** [python-ivi/python-ivi#25]

**Comment by alexforencich** [comment]
No, not at the moment.  The docs also need quite a bit of work.  A good portion of them are generated directly from the python code, but right now most of that has been pulled straight from the IVI spec.  There isn't much about the python-ivi internals, either.
